src/utils/withSuggestions.js

```diff
@@ -24,10 +24,10 @@ function withSuggestions(WrappedComponent) {
     render() {
       const { suggestedData, field } = this.props
       return (
-        <div style={{ display: 'flex', alignItems: 'center' }}>
+        <>
```

Comment:
it's the short syntax for a [React.Fragment](https://reactjs.org/docs/fragments.html). It doesn't render any DOM element. It's used here because the render method needs to return a single component